src/Tests/InstrEngineTests/InstrEngineTests/EmbeddedResourceUtils.cs

```diff
@@ -141,11 +141,13 @@ public static void CleanTestResourceFiles()
             }
         }
 
-        private static Stream GetEmbeddedResource(string fileName, bool required = true)
+        private static Stream GetEmbeddedResource(string fileName, bool required = true, string alternateResourcesPath = null)
```

Comment:
>string alternateResourcesPath = null [](http://example.com/codeflow?start=89&length=36)

You can just do string resourcesPath = EmbeddedResourcePath. That is probably better, so that way we just end up getting a test failure if someone accidentally passes null or empty (same above).